frontend/src/views/Public.vue

```diff
@@ -168,125 +153,90 @@ import {
-      },
-      {
-        name: 'Tom Cook',
-        handle: 'tomcook',
-        imageUrl:
-          'https://images.unsplash.com/photo-1472099645785-5658abf4ff4e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80'
-      }
-    ],
-    totalResponses: 12,
-    lastUpdated: 'March 17, 2020',
-    pinned: true,
-    open: true
-  },
-  {
-    id: 2,
-    title: 'Was eJournal founded during this inbox?',
-    responses: [
-      {
-        name: 'Dries Vincent',
-        handle: 'driesvincent',
-        imageUrl:
-          'https://images.unsplash.com/photo-1506794778202-cad84cf45f1d?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80'
-      },
-      {
-        name: 'Lindsay Walton',
-        handle: 'lindsaywalton',
-        imageUrl:
-          'https://images.unsplash.com/photo-1517841905240-472988babdf9?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80'
-      },
-      {
-        name: 'Courtney Henry',
-        handle: 'courtneyhenry',
-        imageUrl:
-          'https://images.unsplash.com/photo-1438761681033-6461ffad8d80?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80'
-      },
-      {
-        name: 'Tom Cook',
-        handle: 'tomcook',
-        imageUrl:
-          'https://images.unsplash.com/photo-1472099645785-5658abf4ff4e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80'
-      }
-    ],
-    totalResponses: 8,
-    lastUpdated: 'March 18, 2020',
-    pinned: false,
-    open: false
-  }
-]
-const pinnedTickets = tickets.filter((ticket) => ticket.pinned)
+import {
+  LocationMarkerIcon as NotPinnedIcon
+} from '@heroicons/vue/outline'
 
 export default {
   name: 'Public',
   components: {
     ChevronRightIcon,
     DotsVerticalIcon,
-    DuplicateIcon,
-    PencilAltIcon,
-    TrashIcon,
-    UserAddIcon,
+    ExternalLinkIcon,
     Menu,
     MenuButton,
     MenuItem,
-    MenuItems
+    MenuItems,
+    PinnedIcon,
+    NotPinnedIcon
   },
   data: () => ({
-    userInbox: null,
-    side: false
+    is_staff: false
   }),
-  async mounted () {
-    const response = await axios.get(`/api/me/inboxes/${this.$route.params.inboxId}`)
-    this.userInbox = response.data
+  created () {
+    axios.get(`/api/inboxes/${this.$route.params.inboxId}/role`)
+      .then((response) => {
+        this.is_staff = (response.data.key === 'AGENT' || response.data.key === 'MANAGER')
+      })
+
+    this.getTickets()
   },
-  setup () {
-    return {
-      tickets,
-      pinnedTickets
+  methods: {
+    getTickets () {
+      const { inboxId } = this.$route.params
+
+      axios
+        .get(`/api/inboxes/${inboxId}/tickets`, {
+          params: {
+            // q: this.search,
+            // show_personal: this.showPersonal,
+            // labels: labelsIds
```

Comment:
Kan dit weg?

Reply:
Ja, ik wil nog in de toekomst toevoegen dat je kan zoeken op tickets dus wilde even onthouden hoe dat werkt, maar dat weet ik nu wel.